src/tree.rs

```diff
@@ -0,0 +1,21 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+//! Implementation of a Merkle Tree with a Dusk-Poseidon backend and zero-knowledge opening proof powered by PLONK.
```

Comment:
In the previous file 'src/tree/mod.rs' at this point there was a much larger documentation of the tree structure with an example. Is the example no longer valid or is there another reason why you removed?

Reply:
It is duplicated with the `README.md`